api/client/volume/client.go

```diff
@@ -632,6 +644,24 @@ func (v *volumeClient) CloudBackupSchedCreate(
 	return createResponse, nil
 }
 
+// CloudBackupGroupSchedCreate for a volume group creates a schedule to backup
+// volume group to the cloud
+func (v *volumeClient) CloudBackupGroupSchedCreate(
+	input *api.CloudBackupGroupSchedCreateRequest,
+) (*api.CloudBackupSchedCreateResponse, error) {
+	createResponse := &api.CloudBackupSchedCreateResponse{}
+	req := v.c.Post().Resource(api.OsdBackupPath + "/schedgroup").Body(input)
```

Comment:
Define the paths as a constant in api.go and use them here and api/server/volume.go

Reply:
rest of the operations in the same class don' have them defined. so leaving it as it is.

Reply:
Actually the new APIs I added do this. We'll never fix this if we keep adding string everywhere